src/applications/disability-benefits/526EZ/tests/config/veteranInformation.unit.spec.jsx

```diff
@@ -5,26 +5,11 @@ import { mount } from 'enzyme';
 
 import { DefinitionTester } from '../../../../../platform/testing/unit/schemaform-utils.jsx';
 import formConfig from '../../config/form.js';
-import initialData from '../schema/initialData.js';
 
-initialData.prefilled = false;
-
-describe('526EZ veteran information', () => {
+describe.skip('526EZ veteran information', () => {
```

Comment:
Because this page uses a `ui:description` that's a connected component, I think we'll have to mock out the redux store since we can't really test the page without that.

That said, I didn't want to do that in this PR because it's arguably not the right approach. We've stayed away from it to this point and I didn't want to break that pattern without a larger discussion. Mocking the redux store is more of an integration test which...may be what this actually is? Need more clarity around that before making it happen, anyhow.

Reply:
Ticket to re-enable these tests can be found [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/11281).